tests/e2e/e2e_retry_policy_test.go

```diff
@@ -125,8 +125,11 @@ var _ = OSMDescribe("Test Retry Policy",
 					By("A request that will be retried NumRetries times then succeed")
 					time.Sleep(3 * time.Second)
 					result := Td.HTTPRequest(req)
-					stdout, _, err := Td.RunLocal(filepath.FromSlash("../../bin/osm"), "proxy", "get", "stats", clientPod.Name, "--namespace", client)
+					stdout, stderr, err := Td.RunLocal(filepath.FromSlash("../../bin/osm"), "proxy", "get", "stats", clientPod.Name, "--namespace", client)
 					Expect(err).ToNot((HaveOccurred()))
+					if stderr != nil {
```

Comment:
I'm pretty sure `stderr` is never nil even if it's empty, so I'd change this to `if err != nil`.

Reply:
And I think we'll want to check this before `Expect(err).ToNot((HaveOccurred()))` since I think the test will return as soon as that fails.